pojo-generation-core/src/main/java/uk/gov/justice/generation/pojo/generators/EnumGenerator.java

```diff
@@ -71,7 +77,7 @@ private MethodSpec buildToStringMethod() {
                 .addAnnotation(Override.class)
                 .addModifiers(PUBLIC)
                 .returns(String.class)
-                .addStatement("return $L", VALUE_VARIABLE_NAME)
+                .addStatement("return $L", isNumericEnum ? "String.valueOf(".concat(VALUE_VARIABLE_NAME).concat(")") : VALUE_VARIABLE_NAME)
```

Comment:
this looks quite complicated for an inline switch...it might be better as a method that returns a CodeBlock according to the switch and added using .addCode method